chacha20/CHANGELOG.md

```diff
@@ -5,6 +5,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.10.0 (UNRELEASED)
+### Added
+- `rand_core` support ([#333])
+
+### Removed
+- `chacha20_force_neon` cfg attribute ([#361])
+- `std` cfg attribute ([#397])
+
+### Fixed
+- `StreamId` endianness ([#389])
+- `StreamId` `From` impl ([#392])
+
+### Changed
+- Bump `cipher` from `0.4` to `0.5` ([#338])
+- Bump `rand_core` from `0.4` to `0.5` ([#338])
+- Bump MSRV to 1.85 ([#397])
+- Bump edition to 2024 ([#397])
```

Comment:
I would list it as a one entry in the same way as it was done in the other crates. It's also worth to add an entry about relaxation of the MSRV policy.